docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,351 +1,440 @@
-| `ASSET_KEY+`          | Selects an asset and one layer downstream of the asset. Including multiple `+`s will select that number of downstream layers from the asset. Any number of `+`s is supported. [See an example](#specific-downstream). |
+- The Asset Catalog
+- The Alert Policy creation page (when creating an asset alert)
+- The Insights page
 
-## Examples
+:::
 
-The examples in this section use the following asset graph from the [Dagster University Essentials project](https://github.com/dagster-io/project-dagster-university) to demonstrate how to use the selection syntax:
+## Syntax
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/asset-selection-syntax-dag.png)
+### Filters
 
-### Selecting a single asset \{#single-asset}
+Filters allow you to narrow your asset selection using specific criteria.
 
-To select a single asset, use the asset's asset key. This example selects the `taxi_zones_file` asset:
+| Filter | Syntax | Description | Supported views |
+|--------|--------|-------------|-----------------|
+| **Key (exact)** | `key:"my_asset"` | Selects assets with the exact key `my_asset`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Key with one wildcard** | `key:"prefix_*"`| Selects assets whose key starts with `prefix`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Key with multiple wildcards** | `key:"prefix_*_middlefix_*_suffix"` | Selects assets whose key starts with `prefix`, contains `middlefix`, and ends with `suffix`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Tag (exact)** | `tag:"stage"` | Selects assets tagged with `stage`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Tag (with value)** | `tag:"stage"="value"` | Selects assets tagged with `stage` having a specific `value`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Owner** | `owner:"alice"` | Selects assets owned by `alice`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Group** | `group:"team1"` | Selects assets in the group `team1`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Kind** |  `kind:"table"` |  Selects assets of kind `table`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Code location** | `code_location:"repo1"` | Selects assets located in code location `repo1`. | OSS, Dagster+, Dagster+ branch deployments |
+| **Column tag** | `column:tag: "my_tag"` | Selects assets tagged with `my_tag`. | Dagster+ only |
+| **Columns** | `columns: "my_column"` | Selects assets with a column named `my_column`. | Dagster+ only |
+| **Table name** | `table_name: "my_table"` | Selects assets with a table named `my_table`. | Dagster+ only |
+| **Changed in branch** | `changed_in_branch: "my_branch"` | Selects assets changed in a branch named `my_branch`. | Dagster+ branch deployments only |
```

Comment:
change_in_branch is actually an ENUM.
It can have the following values:

```
export enum ChangeReason {
  CODE_VERSION = 'CODE_VERSION',
  DEPENDENCIES = 'DEPENDENCIES',
  METADATA = 'METADATA',
  NEW = 'NEW',
  PARTITIONS_DEFINITION = 'PARTITIONS_DEFINITION',
  REMOVED = 'REMOVED',
  TAGS = 'TAGS',
}
```

We'll likely add more over time

Reply:
This allows you to filter for assets that changed (for a particular reason) in your branch deployment

Reply:
Oh also "ANY" is a possible value (meaning find assets that changed for any reason in the branch deployment).